docs/data/system/experimental-api/configure-the-sx-prop/ChangeTheBehaviorSxProp.js

```diff
@@ -22,9 +22,15 @@ export default function ChangeTheBehaviorSxProp() {
   return (
     <Stack direction="row" gap={1}>
       <ThemeProvider theme={theme}>
-        <Box sx={{ borderRadius: 'sm', border: '1px solid black', p: 4 }} />
-        <Box sx={{ borderRadius: 'md', border: '1px solid black', p: 4 }} />
-        <Box sx={{ borderRadius: 'lg', border: '1px solid black', p: 4 }} />
+        <Box
+          sx={{ borderRadius: 'sm', border: 1, borderColor: 'text.primary', p: 4 }}
```

Comment:
```suggestion
          sx={{ borderRadius: 'sm', border: 1, borderColor: 'divider', p: 4 }}
```
How about using `divider`? It looks more appropriate than `text.primary` for border color.

Reply:
`divider` looks very light in color visually in dark mode docs. I'd rather prefer this. I know `text.primary` doesn't sound good for border color since its primarily for text, so I removed `borderColor` property itself and kept only `border: 1`, which by default picks the `initial` color.